src/main/java/course_project/seaBattle/Main.java

```diff
@@ -0,0 +1,8 @@
+package course_project.seaBattle;
+
+public class Main {
+
+    public static void main(String[] args) {
+        new Game().run();
```

Comment:
Strange behaviour in case of wrong input
![image](https://user-images.githubusercontent.com/58872509/134039559-2cea437e-9de1-4d2d-b561-5fd46041444e.png)

Reply:
opt: No any message in case I fully destroyed opponent's ship

Reply:
Opt: when ship is destroyed, close cells are not marked as already hit

Reply:
Also strange, but okay
![image](https://user-images.githubusercontent.com/58872509/134039988-ef097770-22ff-4741-9275-132c9bab0ade.png)

Reply:
But I do like simplicity of the approach, nice interface and abstractions.

Reply:
Good! There are a few bugs to fix, except that looks good. You could also introduce some abstractions, like abstract classes/interfaces, but it's optional. 

Btw, why do you need fakeField?

Reply:
SeaBattle has the same bugs as before. Please fix

Reply:
Sorry, can't approve with current bugs